src/main/java/com/internship/socialnetwork/model/User.java

```diff
@@ -0,0 +1,36 @@
+package com.internship.socialnetwork.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.OneToMany;
+import lombok.*;
```

Comment:
We should not import all from the Lombok package, only what we use. You can correct it in the next PR